lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -308,6 +308,82 @@ def test_index_no_visible_section_in_chapter(self):
         self.assertNotIn('Problem 1', response.content)
         self.assertNotIn('Problem 2', response.content)
 
+    def _create_global_staff_user_and_course(self):
+        """
+        User will have staff access and creates the course
+        """
+        self.global_staff = GlobalStaffFactory.create()  # pylint: disable=W0201
+        self.client.login(username=self.global_staff.username, password='test')
+        course = CourseFactory.create()
+        return course
+
+    def _create_url_for_enroll_staff(self, course):
+        """
+        creates the courseware url and enroll staff url
+        """
+        activate_block_id = urlencode({'activate_block_id': unicode(course.location)})
```

Comment:
I think it might be nicer to have the `block_id` be a problem or something that isn't necessarily the course, so we can verify that we've been redirected to exactly the page we were trying to go to.